src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -50,7 +51,7 @@
 (defn tabs
   []
   (let [filter-type (rf/sub [:activity-center/filter-type])]
-    [quo2/scrollable-tabs {:size                32
+    [scrollable-view/view {:size                32
```

Comment:
we should use quo2.core and name which is used in figma design

Reply:
is the name of component in figma scrollable-view ?

Reply:
scrollable-view is a namespace that provide functionality that is shared between scrollable-tags and scrollable-tabs. I didnt to avoid duplicate code. The components in figma are tabs and tags and not scrollable-view

Reply:
@ilmotta @J-Son89 any thoughts?

Reply:
hmm, looking into this now. it seems the functionality of scrollable view should be encapsulated in the tabs and tags...

e.g we have `tabs` and `tags` and then also `scrollable-tabs` and `scrollabe-tags` components.


Also if this scrollable view is common between tabs and tags then the namespace depth should show that.
i.e you have this in
 `quo2.components.tags.scrollable-view` however it should be in something like `quo2.components.common.scrollable-view` or  `quo2.components.common.scrollable-view`

Reply:
> As far as I see, the current implementation still leaves a bit of work on the developer to get the "scrollable" behaviour. It should either be a separate component for both tabs and tags.. or both of these components should be configurable with a prop.. e,g scrollable?

I agree. When I first coded it I wanted something flexible given the uncertainty of its usage. Now that we see tags also using it and the solution works, I would prefer if the tabs and tags components simply had a flag to specify if it was scrollable or not as @J-Son89 said.

But, IMO this should be done in a separate PR, because this PR is working fine.

If we open [Design System > Tabs](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System?node-id=57%3A13214&t=ISb34KWjWoEyP193-0), there's no specification about scrollable appearance. I think this needs to be fixed tbh, so something to bring up to designer's attention @jo-mut.

Reply:
@flexsurfer i think @J-Son89 suggestions are very good in this case. It would be better having scrollable-tags/tabs as separate components. I will work on it

Reply:
@ilmotta i agree. I will talk to design on this.

Reply:
@ilmotta @J-Son89 this means the current implementation of tabs and tags components will be replaced by scrollable-tags and scrollable-tabs component with the flag being what makes the difference whether tabs/tags are scrollable or not. Going by that I don't think we would need what we currently have

Reply:
imo:
I don't think you need to rename the tabs and tags components to `scrollable-*`..

Either make tabs and tags scrollable with a prop, so don't change the name of them..

e.g
```
(defn tags [{ scrollable? .....}]
```

```
(defn tabs [{ scrollable? .....}]
```
or 

keep tabs and tags as is and then make 2 new components with the scrollable capabilities:
i.e there is 4 components, tabs and tags and scrollable-tabs and scrollable-tags